TicTacToe/tictactoe.py

```diff
@@ -0,0 +1,51 @@
+list_of_winners_to_ensure_only_one_player_won = []
+
+def column_check(player, column_number):
+    global list_of_winners_to_ensure_only_one_player_won
+    if game_board[0][column_number] == player and game_board[1][column_number] == player and game_board[2][column_number] == player:
+        list_of_winners_to_ensure_only_one_player_won.append(player)
+
+def row_check(player, row_number):
+    global list_of_winners_to_ensure_only_one_player_won
+    if game_board[row_number][0] == player and game_board[row_number][1] == player and game_board[row_number][2] == player:
+        list_of_winners_to_ensure_only_one_player_won.append(player)
+
+def diagonal_check(player):
+    global list_of_winners_to_ensure_only_one_player_won
+    if game_board[0][0] == player and game_board[1][1] == player and game_board[2][2] == player or game_board[2][0] == player and game_board[1][1] == player and game_board[0][2] == player:
+        list_of_winners_to_ensure_only_one_player_won.append(player)
+
+def __main__(input_matrix):
+    global game_board
```

Comment:
no globals.
please refactor this to work with classes.

Reply:
done